js/map.mjs

```diff
@@ -147,6 +147,22 @@ async function initMap(){
             mapContainer.style = "top:0;padding:2px;"
         }
     }
+    //Setting parameters
+    let settingsArray = ["Location", "Nirnroot", "Wayshrine", "NearbyGates"];
```

Comment:
can you put these in ZoomToInitialLocation and rename that function to something like InitMapParametersFromUrl

Reply:
This doesn't appear to work even if its at the very start of the function. I think something between line 167 and 186 where the function is called is causing the map to visually update or something, because the parameters only work after changing a setting manually that forces a redraw. Should I leave it where it is for now or is there something in this init code that can be shuffled around without causing other problems?